### ISSUE TITLE
Different monitor height makes window super long

### ISSUE THREAD

**Issue by 0b01** [TimUntersberger/nog#92]
Here is my screen layout(please ignore 1): 
![image](https://user-images.githubusercontent.com/1768528/88886018-7b57e180-d1ee-11ea-8790-6c53b40785cc.png)

When using VS Code, the window is super long as you can see in this screenshot:
![image](https://user-images.githubusercontent.com/1768528/88886158-b6f2ab80-d1ee-11ea-94ad-85d0d43e47f3.png)

**Comment by 0b01** [comment]
I don't know the cause for this but could it be a rounding error when calculating the bottom right coordinate of the window?

**Comment by TimUntersberger** [comment]
I tried to replicate this with my own 1920x1080 monitors and it seems to work for me. Are you monitors of different size?

**Comment by 0b01** [comment]
Yes. Different size monitors.

I really want to use wwm because it's the best I've tried so far and it's written in Rust.

**Comment by TimUntersberger** [comment]
Can you try running wwm only on your left (the horizontal one) monitor? This might be a DPI problem.

**Comment by 0b01** [comment]
@TimUntersberger I set multi-monitor to false and got the same issue.

**Comment by TimUntersberger** [comment]
Can you tell which monitor you are using?

**Comment by 0b01** [comment]
Asus predator ultrawide. 3440x1440 iirc

**Comment by TimUntersberger** [comment]
Since this is a DPI problem I'll wait until I have refactored from pixels to percentages. Basically once the resizing issue is closed, everything should be in place to fix this.

**Comment by 0b01** [comment]
VSCode:
![image](https://user-images.githubusercontent.com/1768528/89113628-88f8ab80-d428-11ea-8c1c-04be7f82c899.png)

Notepad:
![image](https://user-images.githubusercontent.com/1768528/89113638-a62d7a00-d428-11ea-9a6b-274275745121.png)

**Comment by TimUntersberger** [comment]
Did you add a rule that tells wwm that vs code has a custom titlebar?

**Comment by 0b01** [comment]
I just use the default which I think contains that line.

**Comment by TimUntersberger** [comment]
I think I fixed it! My initial assumption was right, it was related to DPI, but in a weird way.

To get system metrics, like how big the border of a window is, there exist two functions. One is DPI aware and the other isn't. I am now using the DPI aware function with hopefully the correct DPI.

There exist 4 different ways of getting the DPI of a monitor so I hope I chose the correct one.

The latest dev build contains my attemp at fixing this.
